css/account.css

```
@@ -171,3 +165,48 @@ body {
   font-weight: 500;
   font-size: 1.4rem;
 }
+
+/* 태블릿과 PC에서는 동일한 스타일을 유지하므로 별도 스타일 변경 없음 */
+/* 아이패드 에어를 기준으로 구현하였습니다. (820 x 1180) */
+@media (min-width: 768px) and (max-width: 1199px) {
+}
```

💊 제안
미디어쿼리 조건문에서 min-width를 빼도 동일적용됩니다~
`max-width: 767px`를 조건으로 가진 모바일 미디어 쿼리문이 하단에 위치하기 때문에
해당 조건이 달성되면 모바일 미디어 쿼리문이 우선 적용됩니다.
또한 반응형 작업시 대부분의 경우 PC의 CSS 들을 Tablet에서 상속받고 앞의 속성들을 Mob에서 상속받는 것이
중복작성을 줄일 가능성이 크니 아래처럼 작성하시는 것을 추천드려요~

```suggestion
/* 태블릿과 PC에서는 동일한 스타일을 유지하므로 별도 스타일 변경 없음 */
/* 아이패드 에어를 기준으로 구현하였습니다. (820 x 1180) */
@media (max-width: 1199px) {
}
```